docs/rst/knora-api-server/design-documentation/authentication.rst

```diff
@@ -26,146 +26,139 @@ Authentication in the Knora API Server
+sent with each request. Normally, a web browser does this automatically, i.e. sends the cookie on every request. The
+session token is used by the server to retrieve the user profile. If successful, the user is deemed authenticated.
+
+To **logout** the client can call the same route and provide the logout parameter **/v1/session?logout**. This will
+invalidate the session token and return headers for removing the cookie on the client.
+
+
+Submitting Credentials
 ^^^^^^^^^^^^^^^^^^^^^^
 
-To check the credentials and create a 'session', e.g., by a login window
-in the client, there is a special route called **/v1/authentication**,
-which returns following for each case:
+For **login**, credentials in form of *email* and *password* need to be sent with the request.
 
-**Credentials correct:**
+There are two possibilities to do so:
+ - in the URL submitting the parameters ``email`` and ``password`` (e.g., http://knora-host/v1/resources/resIri?email=userUrlEncodedEmail&password=pw)
+ - in the HTTP authorization header (`HTTP basic authentication`_) when doing a HTTP request to the API
+   When using Python's module ``requests``, the credentials can simply be submitted as a tuple with each request using
+   the param ``auth`` (`python requests`_).
 
-::
+An alternative way for accessing all routes is to simply supply the *email* and *password* credentials on each request
+either as URL parameters or in the HTTP authorization header.
 
-    {
-      "status": 0,
-      "message": "credentials are OK".
-      "sid": "1437643844783"
-    }
 
-In this case, the found user profile is written to a cache and stored
-under the ''sid'' key. Also a header requesting to store the ''sid'' in
-a cookie is sent. On subsequent access to all the other routes, the
-''sid'' is used to retrieve the cached user profile. If successful, the
-user is deemed authenticated.
+Checking Credentials
+^^^^^^^^^^^^^^^^^^^^
 
-**Email wrong:**
+To check the credentials, there is a special route called **/v1/authenticate**, which can be used to check if the
+credentials are valid.
 
-::
+Usage Scenarios
+^^^^^^^^^^^^^^^
 
-    {
-      "status": 2,
-      "message": "bad credentials: user not found"
-    }
+1. Create session by logging-in, send session token on each subsequent request, and logout when finished.
+2. Send *email*/*password* credentials on every request.
 
-**Password wrong:**
 
-::
+Usage V2
+--------
 
-    {
-      "status": 2,
-      "message": "bad credentials: user found, but password did not match"
-    }
+Login and Logout
+^^^^^^^^^^^^^^^^
 
-**No credentials:**
+A client sends a POST request (``{"email":"usersemail", "password":"userspassword"}``) to the
+**/v2/authentication** route with *email* and *password* in the body. If the credentials are valid,
+a `JSON WEB Token`_ (JWT) will be sent back in the response. On all subsequent calls to any route,
```

Comment:
Could you add an example of what a response containing a JWT looks like?

Reply:
added.